test.js

```diff
@@ -1,7 +1,12 @@
-'use strict';
-var assert = require('assert');
-var splitLines = require('./');
+import test from 'ava';
+import fn from './';
```

Comment:
Yes I know because we had a discussion on this before but it seemed that not everybody like it. Sindre, Kevva and I always use `m` in our modules though.